compiler/rustc_error_codes/src/error_codes/E0660.md

```diff
@@ -9,4 +9,4 @@ llvm_asm!("nop" "nop");
 Considering that this would be a long explanation, we instead recommend you
 take a look at the [`llvm_asm`] chapter of the Unstable book:
 
-[llvm_asm]: https://doc.rust-lang.org/stable/unstable-book/library-features/llvm-asm.html
+[`llvm_asm`]: https://doc.rust-lang.org/stable/unstable-book/library-features/llvm-asm.html
```

Comment:
Why this change?

Reply:
The link was broken before. The reference text has to match _exactly_ or it won't be treated as a link.

Reply:
https://github.com/rust-lang/rust/issues/62345

Reply:
Oh you're right. Nevermind then!